### PR TITLE
[Zoom] Update existing scopes with granular scopes

### PR DESCRIPTION
**Pull request by moxarth-rathod**
## Part of https://github.com/elastic/connectors/issues/2818

Existing permissions are deprecated in the Zoom developer app, replacing them with the new granular scopes - https://developers.zoom.us/docs/integrations/oauth-scopes-overview/. 
Fyi, the `cloud_recording:read:list_user_recordings:admin` scope is not tested due to not having a pro account.

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/connector/docs/connectors-zoom.asciidoc [hunk 1]
```diff
@@ -63,17 +63,18 @@ To connect to Zoom you need to https://developers.zoom.us/docs/internal-apps/s2s
...
-chat_message:read:admin
-report:read:admin
+user:read:list_users:admin
+meeting:read:list_meetings:admin
+meeting:read:list_past_participants:admin
+cloud_recording:read:list_user_recordings:admin
+team_chat:read:list_user_channels:admin
+team_chat:read:list_user_messages:admin
 ----
+*Note:* the connector requires a minimum scope of `user:read:list_users:admin` to ingest data into Elasticsearch.
```
Comment (leemthompo) [inline docs/reference/connector/docs/connectors-zoom.asciidoc:77]:
```suggestion
[NOTE]
====
The connector requires a minimum scope of `user:read:list_users:admin` to ingest data into Elasticsearch.
====
```

asciidoc syntax ;)

File: docs/reference/connector/docs/connectors-zoom.asciidoc [hunk 2]
```diff
@@ -220,17 +221,18 @@ To connect to Zoom you need to https://developers.zoom.us/docs/internal-apps/s2s
...
```
Comment (leemthompo) [inline docs/reference/connector/docs/connectors-zoom.asciidoc:235]:
```suggestion
[NOTE]
====
The connector requires a minimum scope of `user:read:list_users:admin` to ingest data into Elasticsearch.
====
```

**Review by leemthompo — APPROVED** [review]
Thanx! Just one minor syntax fix required 🚀

**Comment by leemthompo** [comment]
@moxarth-elastic few minor process points for future PRs in the Elasticsearch repo:

- Try to preface PR title with `[DOCS][Connectors] ...`
- Always specify what versions are concerned in the PR description, because we might need to cherrypick changes to the previous versions which live in the private repo
- Delete the checklist from the PR description to make it easier to read

**Comment by moxarth-elastic** [comment]
> @moxarth-elastic few minor process points for future PRs in the Elasticsearch repo:
> 
> * Try to preface PR title with `[DOCS][Connectors] ...`
> * Add the `>docs` label
> * Request reviews from writers and devs
> * Always specify what versions are concerned in the PR description, because we might need to cherrypick changes to the previous versions which live in the private repo
> * Delete the checklist from the PR description to make it easier to read

Noted, Thanks! 
And FYI, i don't enough access to add labels in the PR.

**Comment by moxarth-elastic** [comment]
@leemthompo can you please re-run the CI and merge the PR? I don't have merge access.

**Comment by leemthompo** [comment]
@elasticmachine update branch

**Comment by leemthompo** [comment]
@moxarth-elastic how many Elastic versions should this be backported to?

File: docs/reference/connector/docs/connectors-zoom.asciidoc [hunk 3]
```diff
@@ -63,17 +63,21 @@ To connect to Zoom you need to https://developers.zoom.us/docs/internal-apps/s2s
...
+meeting:read:list_meetings:admin
+meeting:read:list_past_participants:admin
+cloud_recording:read:list_user_recordings:admin
+team_chat:read:list_user_channels:admin
+team_chat:read:list_user_messages:admin
 ----
+[NOTE]
+====
+The connector requires a minimum scope of `user:read:list_users:admin` to ingest data into Elasticsearch.
+====
```
Comment (leemthompo) [inline docs/reference/connector/docs/connectors-zoom.asciidoc:81]:
```suggestion
+
```

File: docs/reference/connector/docs/connectors-zoom.asciidoc [hunk 4]
```diff
@@ -220,17 +224,21 @@ To connect to Zoom you need to https://developers.zoom.us/docs/internal-apps/s2s
...
```
Comment (leemthompo) [inline docs/reference/connector/docs/connectors-zoom.asciidoc:242]:
```suggestion
+
```

**Comment by leemthompo** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[](https://github.com/elastic/elasticsearch/pull/114238)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by moxarth-elastic** [comment]
> @moxarth-elastic how many Elastic versions should this be backported to?

we can do it for 8.11.0 and later versions - https://github.com/elastic/connectors/pull/1504.

**Pull request merged**